tests/integration/cli/projects/with-contracts/ape-config.yaml

```diff
@@ -7,3 +7,7 @@ dependencies:
 test:
   # `false` because running pytest within pytest.
   disconnect_providers_after: false
+
+ethereum:
+  compilers:
+    - ethpm
```

Comment:
take care to include newlines at the end of all files

Reply:
this is not visible on github but at least now they have new lines.